packages/template-retail-react-app/app/pages/checkout/partials/shipping-options.jsx

```diff
@@ -210,6 +232,7 @@ export default function ShippingOptions() {
                                     textDecoration="line-through"
                                     color="gray.600"
                                     marginLeft={1}
+                                    aria-hidden="true"
```

Comment:
set `aria-hidden="true"` so that the screen reader doesn't repeat the price as it would add confusion